app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -59,6 +60,11 @@ open class HomeActivity : AppCompatActivity() {
         handleOpenedFromExternalSourceIfNecessary(intent)
     }
 
+    override fun onResume() {
+        super.onResume()
+        components.analytics.metrics.track(Event.OpenedApp)
```

Comment:
Does this have a built-in opt out? How do users turn telemetry off?

Reply:
Currently we disable telemetry in settings